### ISSUE TITLE
Confused about the training loss used in pretrain

### ISSUE THREAD

**Issue by XiejiLi** [MIV-XJTU/FLAME#5]
Dear authors,
  Thank you for your open source code, it help our research alot.
  One thing I am confused is that the use of mp-sigmoid param during training. It seems you are using a sigmoid multi-positve text learning stratagy during pretrain stage. Your training documentation shows the `mp-sigmoid` flag is on.
 You can refer this line: https://github.com/MIV-XJTU/FLAME/blob/main/Training.md?plain=1#L45
 But it obviously not matching with the formula you provided in your paper.

<img width="661" alt="Image" src="https://github.com/user-attachments/assets/904339a4-5f02-4b25-b86e-00deb4359c3b" />

  Is it a wrong typo in your training documentation?

Thanks again, have a good day.

**Comment by CAOANJIA** [comment]
Hi~

You can use --mp-nce.

In theory, any multi-positive loss can be used. We assume that sigmoid loss is better than infonce loss, so --mp-sigmoid is recommended and currently set by default (although we have tested that the overall results of the two are very close, infonce seems to be slightly better for classification, while sigmoid seems to be slightly better for retrieval? But based on the comparison between SigLIP and CLIP, we expect sigmoid loss to have the potential for scalability~).

**Comment by XiejiLi** [comment]
Got it! Thank you so much for sharing your observation and experiment conclusion. It help us alot!

Best wish.
